org.eclipse.xtext.xbase.testdata/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,6 @@ Require-Bundle: org.eclipse.xtext.ide,
  org.antlr.runtime;bundle-version="[3.2.0,3.2.1)"
 Automatic-Module-Name: org.eclipse.xtext.xbase.ide
 Export-Package: .,
- bar,
```

Comment:
I'm curious: Why did you remove / change the test case?

Reply:
As far as I understand the purpose of the serialize method that basically means, that only import statements are filtered now, while an a potentially qualification of a reference can still occur.

Reply:
Are you sure? I removed  org.eclipse.xtext.xbase.testdata/src/bar/String.java

Reply:
did you forget to push the delete with this PR?

Reply:
![grafik](https://user-images.githubusercontent.com/75562299/101885520-84908c00-3b9a-11eb-9a70-e893198b002e.png)

Reply:
must be blind. sry

Reply:
no problem 🙂

Reply:
@szarnekow If you prefer, I can re-add the second test case again, of course.

Reply:
@nineninesevenfour Yes, that would be good.